diff -Naur -x .git lge-kernel-kang/drivers/mmc/host/msm_sdcc.c lge-kernel-faux123/drivers/mmc/host/msm_sdcc.c
--- lge-kernel-kang/drivers/mmc/host/msm_sdcc.c	2012-01-13 11:41:25.191758587 +0100
+++ lge-kernel-faux123/drivers/mmc/host/msm_sdcc.c	2012-01-09 18:58:33.682069933 +0100
@@ -48,7 +48,7 @@
 #define DRIVER_NAME "msm-sdcc"
 
 static unsigned int msmsdcc_fmin = 144000;
-static unsigned int msmsdcc_fmax = 49152000;
+static unsigned int msmsdcc_fmax = 50000000;
 static unsigned int msmsdcc_4bit = 1;
 static unsigned int msmsdcc_pwrsave = 1;
 static unsigned int msmsdcc_piopoll = 1;
